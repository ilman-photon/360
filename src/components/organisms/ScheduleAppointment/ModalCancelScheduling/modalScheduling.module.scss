.modalDialog {
  width: auto;
  @media (max-width: 556px) {
    width: 100% !important;
    margin: 0;
  }
}

.boxModalContents {
  margin: 16px;
  width: 635px !important;
  @media (max-width: 556px) {
    width: auto !important;
  }

  .scheduledText {
    font-weight: 500;
    font-size: 22px;
    line-height: 30px;
    letter-spacing: 0.0016em;
    color: #003b4a;
  }

  form {
    display: grid;
  }

  .checkBoxContainer {
    padding: 0px;
    overflow-y: visible;
  }

  .formButton {
    display: flex;
    justify-content: center;
    align-items: center;
    padding: 13px 20px;
    border-radius: 30px;
    font-size: 14px;
    height: 40px;
    width: 100%;
    text-transform: none;
    font-weight: 600;
    font-family: "Libre Franklin", sans-serif;
    font-style: normal;
    line-height: 18px;
    box-shadow: none !important;

    &.cancel {
      background-color: #b93632 !important;
      border-radius: 30px;
      text-transform: none;
      white-space: nowrap;
    }
    &.outlined {
      background-color: transparent !important;
      border: solid 1px #007e8f;
      color: #007e8f;
      border-radius: 30px;
      text-transform: none;
    }
  }
}
