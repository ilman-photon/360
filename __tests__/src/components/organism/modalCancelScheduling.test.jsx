import { render, fireEvent, waitFor } from "@testing-library/react";
import "@testing-library/jest-dom";
import ModalCancelScheduling from "../../../../src/components/organisms/ScheduleAppointment/ModalCancelScheduling/modalCancelScheduling";

describe("App", () => {
  let container;
  const mockCallBack = jest.fn();
  beforeEach(() => {
    container = render(
      <ModalCancelScheduling
        isOpen={true}
        OnClickCancel={mockCallBack}
        OnCancelClicked={mockCallBack}
      />
    );
  });

  it("ModalCancelScheduling UI", async () => {
    await waitFor(() => container.getByText(/cancelTitle/i));

    expect(container.getByText(/cancelReason/i)).toBeInTheDocument();

    const otherRadio = container.getByRole("radio", { name: /Other/i });
    fireEvent.click(otherRadio);
    expect(otherRadio.value).toEqual("other");

    const otherField = container.getByLabelText(/cancelOther/i);
    fireEvent.change(otherField, { target: { value: "Reason" } });
    expect(otherField.value).toEqual("Reason");

    const cancelButton = container.getByRole("button", { name: /btnCancel/i });
    const KeepButton = container.getByRole("button", { name: /btnKeep/i });
    expect(KeepButton).toBeTruthy();
    fireEvent.click(cancelButton);
  });
});
