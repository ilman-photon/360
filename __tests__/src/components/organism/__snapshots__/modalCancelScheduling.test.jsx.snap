// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App ModalCancelScheduling render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    style="padding-right: 1024px; overflow: hidden;"
  >
    <div
      aria-hidden="true"
    />
    <div
      class="MuiDialog-root modalDialog MuiModal-root css-1ulhvbw-MuiModal-root-MuiDialog-root"
      role="presentation"
    >
      <div
        aria-hidden="true"
        class="MuiBackdrop-root css-yiavyu-MuiBackdrop-root-MuiDialog-backdrop"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
      />
      <div
        data-test="sentinelStart"
        tabindex="0"
      />
      <div
        class="MuiDialog-container MuiDialog-scrollPaper css-hz1bth-MuiDialog-container"
        role="presentation"
        style="opacity: 1; webkit-transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms; transition: opacity 225ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
        tabindex="-1"
      >
        <div
          aria-describedby="alert-dialog-description"
          aria-labelledby="customized-dialog-title"
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation24 MuiDialog-paper MuiDialog-paperScrollPaper MuiDialog-paperWidthSm css-1t1j96h-MuiPaper-root-MuiDialog-paper"
          role="dialog"
        >
          <div
            class="boxModalContents MuiBox-root css-68zbsl"
          >
            <span
              class="MuiTypography-root MuiTypography-bodyMedium scheduledText css-37jdci-MuiTypography-root"
            >
              cancelTitle
            </span>
            <div
              class="MuiDialogContent-root checkBoxContainer css-ypiqx9-MuiDialogContent-root"
            >
              <div
                class="MuiBox-root css-0"
              >
                <form
                  style="display: grid;"
                >
                  <div
                    class="MuiFormControl-root css-xgbsib-MuiFormControl-root"
                    label="cancelReason"
                    options="[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]"
                    rules="[object Object]"
                  >
                    <label
                      aria-label="cancelReason"
                      class="MuiFormLabel-root MuiFormLabel-colorPrimary css-p3vhbq-MuiFormLabel-root"
                      id="row-radio-buttons-group-label"
                      tabindex="0"
                    >
                      cancelReason
                    </label>
                    <div
                      aria-labelledby="row-radio-buttons-group-label"
                      class="MuiFormGroup-root MuiFormGroup-row css-1toi8sz-MuiFormGroup-root"
                      role="radiogroup"
                    >
                      <label
                        aria-label="Family emergency Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="familyemergency"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Family emergency
                        </span>
                      </label>
                      <label
                        aria-label="Patient passed away Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="passedAway"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Patient passed away
                        </span>
                      </label>
                      <label
                        aria-label="Family emergency Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="familyEmergency"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Family emergency
                        </span>
                      </label>
                      <label
                        aria-label="Not covered by insurance Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="notCovered"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Not covered by insurance
                        </span>
                      </label>
                      <label
                        aria-label="Rescheduled appointment Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="rescheduled"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Rescheduled appointment
                        </span>
                      </label>
                      <label
                        aria-label="Patient hospitalized Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="PatientHospitalized"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Patient hospitalized
                        </span>
                      </label>
                      <label
                        aria-label="Patient illness Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="patientIllness"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Patient illness
                        </span>
                      </label>
                      <label
                        aria-label="Transportation issues Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="transportationIssues"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Transportation issues
                        </span>
                      </label>
                      <label
                        aria-label="Weather Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="weather"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Weather
                        </span>
                      </label>
                      <label
                        aria-label="Homebound Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="homebound"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Homebound
                        </span>
                      </label>
                      <label
                        aria-label="Other Radio Button"
                        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-1hsda4-MuiFormControlLabel-root"
                      >
                        <span
                          class="MuiRadio-root MuiRadio-colorPrimary MuiButtonBase-root MuiRadio-root MuiRadio-colorPrimary PrivateSwitchBase-root css-hvpvox-MuiButtonBase-root-MuiRadio-root"
                        >
                          <input
                            class="PrivateSwitchBase-input css-1m9pwf3"
                            name="row-radio-buttons-group"
                            type="radio"
                            value="other"
                          />
                          <span
                            class="css-hyxlzm"
                          >
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hbvpl3-MuiSvgIcon-root"
                              data-testid="RadioButtonUncheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
                              />
                            </svg>
                            <svg
                              aria-hidden="true"
                              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1hhw7if-MuiSvgIcon-root"
                              data-testid="RadioButtonCheckedIcon"
                              focusable="false"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M8.465 8.465C9.37 7.56 10.62 7 12 7C14.76 7 17 9.24 17 12C17 13.38 16.44 14.63 15.535 15.535C14.63 16.44 13.38 17 12 17C9.24 17 7 14.76 7 12C7 10.62 7.56 9.37 8.465 8.465Z"
                              />
                            </svg>
                          </span>
                          <span
                            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                          />
                        </span>
                        <span
                          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-ahj2mt-MuiTypography-root"
                        >
                          Other
                        </span>
                      </label>
                    </div>
                    <p
                      class="MuiFormHelperText-root MuiFormHelperText-sizeMedium MuiFormHelperText-contained css-1wc848c-MuiFormHelperText-root"
                    />
                  </div>
                  <div
                    class="css-1g80z4v-MuiStack-root"
                  >
                    <button
                      class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root formButton cancel css-6o0hah-MuiButtonBase-root-MuiButton-root"
                      data-testid="loc_btnCancel"
                      tabindex="0"
                      type="submit"
                    >
                      btnCancel
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                    <button
                      class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root formButton outlined css-1ygdobz-MuiButtonBase-root-MuiButton-root"
                      data-testid="loc_btnKeep"
                      tabindex="0"
                      type="button"
                    >
                      btnKeep
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        data-test="sentinelEnd"
        tabindex="0"
      />
    </div>
  </body>,
  "container": <div
    aria-hidden="true"
  />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
